### PR TITLE
add description of OGS

### PR DESCRIPTION
**Pull request by syuhei176**

### CODE REVIEW

File: pages/docs/introduction.mdx [hunk 1]
```diff
@@ -8,6 +8,8 @@ Gazelle is a scaling technology for blockchain based on Plasma and Optimistic Vi
 
 Plasma is a second layer scaling solution which introduces **transaction aggregator** who recieves all the transactions happend off-chain and generate a merkle tree with all the transactions' hash as leaves of the tree. By submitting root of the tree to the main-chain periodically, all the users are assured to protect their assets on Plasma. You can find detailed discussion about plasma <a href="https://www.plasma.io/plasma.pdf" target="_blank" rel="noopener">here</a>
 
+Optimistic Virtual Machine(OVM) is the abstraction of Layer 2 dispute logic, designed to support all protocols using fraud-proof. Dispute logic is written by Predicate logic as property. Every property is claimed to the same smart contract called Adjudication Contract. They can prove as true or false under dispute logic.
```
Comment (daiki44) [inline pages/docs/introduction.mdx:11]: @syuhei176 The sentence is good!
Will you create a diagram?

Comment (syuhei176) [reply pages/docs/introduction.mdx:11]: No, I won't. What kind of diagram is good?

One idea is the diagram that Plasma aggregator aggregates transactions and submit Merkle root to blockchain. However it's little bit difficult to make clear difference from Side-chain.

And I think the diagram about OVM is too abstract to draw concrete diagram.

Comment (daiki44) [reply pages/docs/introduction.mdx:11]: > One idea is the diagram that Plasma aggregator aggregates transactions and submit Merkle root to blockchain.

Sounds great.
At this point, the users should be able to understand the abstract.
What do you think? 

> However it's little bit difficult to make clear difference from Side-chain.

I guess it depends on the target reader.
It is enough if the target is a beginner.

> And I think the diagram about OVM is too abstract to draw concrete diagram.

I agree.


Let's talk about this topic tomorrow!

Comment (tkmct) [inline pages/docs/introduction.mdx:11]: > support all protocols

How do you define **all protocols**?

> as property

I don't think general noun property makes sense to for those who don't know OVM at all. What about using a word **proposition**?

> They can prove as true

can be proven?

Comment (tkmct) [reply pages/docs/introduction.mdx:11]: Also what about mention how the claimed proposition will be decided on the Adjudication Contract little bit more?

Comment (syuhei176) [reply pages/docs/introduction.mdx:11]: proposition is very good. will think it over

Comment (syuhei176) [reply pages/docs/introduction.mdx]: added how the claimed proposition will be decided and benefit of OGS.

**Pull request merged**